site/content/entry/yuzu-progress-report-jun-2020/index.md

```diff
@@ -81,9 +81,9 @@ By implementing a [missing texture format](https://github.com/yuzu-emu/yuzu/pull
     "./xcde_after2.png"
   >}}
   
-Intel used to have a bug in their OpenGL driver causing many games using 'Compute Shaders' to crash or render effects incorrectly, forcing us to disable them on any supported Intel GPU and, by doing so, affecting many games. One game of note is `Super Smash Bros. Ultimate`, where compute shaders are used to render character model animations and without them, character models will be in a `T-pose`.
+Intel used to have a bug in their OpenGL driver causing many games using `Compute Shaders` to crash or render effects incorrectly. This forced us to disable them on any supported Intel GPU, which introduced adverse effects in many games. One game of note is `Super Smash Bros. Ultimate`, wherein compute shaders are used to render character model animations and without them, character models will be in a `T-pose`.
 
-Additionally, Intel's OpenGL `Geometry Shader` implementation, in contrast to AMD's and NVIDIA's one, is too rigorous - following the official OpenGL specification to a T due to the lack of an OpenGL extension called `GL_ARB_shader_viewport_layer_array`. This forces us to re-evaluate yuzu's implementation of it.
+Additionally, Intel's OpenGL `Geometry Shader` implementation (in contrast to AMD's and Nvidia's one) is too rigorous, as it follows the official OpenGL specification to a T, therefore lacking an OpenGL extension called `GL_ARB_shader_viewport_layer_array`. This forces us to re-evaluate yuzu's implementation of it.
```

Comment:
Here it was intended to mean that it lacks an extension that enables additional functionality. So Intel is forced to follow what the spec suggests without this functionality.

Reply:
Thanks for the insight